Source/WebCore/page/Quirks.h

```diff
@@ -170,6 +170,9 @@ class Quirks {
     void setNeedsConfigurableIndexedPropertiesQuirk() { m_needsConfigurableIndexedPropertiesQuirk = true; }
     bool needsConfigurableIndexedPropertiesQuirk() const;
 
+    bool needsToCopyUserSelectNoneQuirk() const { return m_needsToCopyUserSelectNoneQuirk; }
```

Comment:
Could you add the reference to the bug in Webkit or radar? So it's easier to maintain when we need to remove it. Thanks.

Reply:
As an example you can check the modifications on Quirks.cpp

Reply:
Sure, although that's what blame is for.